devices/salus_controls.js

```diff
@@ -107,6 +107,7 @@ module.exports = [
     {
         zigbeeModel: [
             'SS909ZB',
+            'PS600',
             'PS600\u0000\u0000 �\u0001 ��\u0000\u0000\t\u0001@\u0011�u\u0000\u0000\u0018&\n\u0005\u0000B',
             'PS600\u0000\u0000 �\u0001 ��\u0000\u0000\t\u0001@\u0011�u\u0000\u0000\u0018\u0006\n\u0005\u0000',
             'PS600\u0000\u0000 �\u0001 ��\u0000\u0000\t\u0001@\u0011�u\u0000\u0000\u0018\u0006\n\u0005\u0000B',
```

Comment:
Could these lines now be removed?

Reply:
Yep, I wanted to make sure we didn’t break anything first and it looks like we didn’t! Should be all gone now!